src/app/app.module.ts

```diff
@@ -4,7 +4,7 @@ import { FormsModule } from "@angular/forms";
 import { BrowserModule, HammerModule } from "@angular/platform-browser";
 import { BrowserAnimationsModule } from "@angular/platform-browser/animations";
 import {
-    IgxAutocompleteModule, IgxButtonModule, IgxDropDownModule,
+    IgxAutocompleteModule, IgxButtonModule, IgxDateTimeEditorModule, IgxDropDownModule,
```

Comment:
IgxDateTimeEditorModule should not exist in app.module.ts as this is related to the samples browser app only.